### PR TITLE
Add preset BAMs including GIAB HG008 tumor data.

### PR DESCRIPTION
**Pull request by MariaNattestad**
1. Add BAM preset list.
2. Fix a bam loading error for when no reads are found when fetching from a locus.
3. Move _CLI variable and initialization to file_parsing.js so both Ribbon and SplitThreader JS scripts can use it regardless of which is loaded first.

### CODE REVIEW

File: js/file_parsing.js [hunk 1]
```diff
@@ -139,3 +160,21 @@ function parseBamReads(raw) {
       return record;
     });
 }
+
+// ===========================================================================
+// == Biowasm / Aioli
```
Comment (robertaboukhalil) [inline js/file_parsing.js:165]: Great idea!

File: js/index_ribbon.js [hunk 2]
```diff
@@ -6435,6 +6438,7 @@ function parse_bam_record(record) {
 }
 
 function use_fetched_data(records) {
+  // records is [undefined]
```
Comment (robertaboukhalil) [inline js/index_ribbon.js:6441]: Left over comment?

File: js/index_ribbon.js [hunk 3]
```diff
@@ -6541,6 +6545,51 @@ function submit_bam_url() {
 }
 d3.select("#submit_bam_url").on("click", submit_bam_url);
 
+const _bam_presets = [
+  {
+    url: "https://42basepairs.com/download/gs/deepvariant/pacbio-case-study-testdata/HG003.pfda_challenge.grch38.phased.bam",
+    name: "HG003 PacBio phased",
+    "42basepairs_url":
+      "https://42basepairs.com/browse/gs/deepvariant/pacbio-case-study-testdata?file=HG003.pfda_challenge.grch38.phased.bam",
+  },
```
Comment (robertaboukhalil) [inline js/index_ribbon.js:6554]: You could infer `42basepairs_url` by replacing `download` with `browse`

**Review by robertaboukhalil — APPROVED** [review]
Looks good!

File: js/index_ribbon.js (hunk 3 above)
Comment (MariaNattestad) [reply js/index_ribbon.js:6553]: Yea true, but it's just nice to be able to check the URLs here by clicking on them, also more foolproof and future-proof.

**Pull request merged**